lib/gui/app/pages/finish/controllers/finish.js

```diff
@@ -57,10 +59,20 @@ module.exports = function ($state) {
       selectionState.deselectImage()
     }
     selectionState.deselectAllDrives()
-    analytics.logEvent('Restart', options)
+    analytics.logEvent('Restart', _.assign({
```

Comment:
Can we use `Object.assign()` here instead?

Reply:
The linter prefers `_.assign` over `Object.assign`... :/

Reply:
Ugh. I thought we had already changed that setting...

Reply:
I guess we'll keep that for now then; and have another 👀 at the linter settings.